### ISSUE TITLE
Create a custom implementation for Dropdown

### ISSUE THREAD

**Issue by kapraran** [kapraran/vu-debug-gui#14]
It would be really nice if someone would like to create a tweakpane plugin to support custom dropdown menu 😄 .
It should be a whole different repo and then I'll just use it in this lib too.

For now i may try to use this one with some changes
https://github.com/donmccurdy/tweakpane-plugin-thumbnail-list

**Comment by IllustrisJack** [comment]
Is the list-blade the one that doesn´t work properly? ):

**Comment by kapraran** [comment]
yeah, it uses vanilla 'select' under the hood and VU doesn't like that

**Comment by IllustrisJack** [comment]
NoFate is currently updating CEF to a newer version for Venice Unleashed. Maybe that fixes the native dropdown (who knows lol)
